docs/requirements.txt

```diff
@@ -1,3 +1,4 @@
-cython>=0.25
-sphinx>=1.7
+cython>=0.29
 gevent
+tornado
```

Comment:
Why was tornado added to the deps list?

Reply:
Thanks for your comment.

Without this, importing zmq.auth fails. And, test-requirements.txt also lists tornado.

That’s why I added it.

Reply:
<details>
<summary>FYI: What would occur if there was no tornado.</summary>

```
Running Sphinx v3.0.4
loading translations [en]... done
making output directory... done
loading intersphinx inventory from https://docs.python.org/3/objects.inv...
loading intersphinx inventory from http://www.tornadoweb.org/en/stable/objects.inv...
intersphinx inventory has moved: http://www.tornadoweb.org/en/stable/objects.inv -> https://www.tornadoweb.org/en/stable/objects.inv
building [mo]: targets for 0 po files that are out of date
building [readthedocs]: targets for 29 source files that are out of date
updating environment: [new config] 29 added, 0 changed, 0 removed
reading sources... [  3%] api/index
reading sources... [  6%] api/zmq
reading sources... [ 10%] api/zmq.asyncio
reading sources... [ 13%] api/zmq.auth
reading sources... [ 17%] api/zmq.auth.ioloop
reading sources... [ 20%] api/zmq.auth.thread
reading sources... [ 24%] api/zmq.decorators
reading sources... [ 27%] api/zmq.devices
reading sources... [ 31%] api/zmq.eventloop.future
reading sources... [ 34%] api/zmq.eventloop.ioloop
reading sources... [ 37%] api/zmq.eventloop.zmqstream
reading sources... [ 41%] api/zmq.green
reading sources... [ 44%] api/zmq.log.handlers
reading sources... [ 48%] api/zmq.ssh.tunnel
reading sources... [ 51%] api/zmq.utils.jsonapi
reading sources... [ 55%] api/zmq.utils.monitor
reading sources... [ 58%] api/zmq.utils.win32
reading sources... [ 62%] api/zmq.utils.z85
reading sources... [ 65%] changelog
reading sources... [ 68%] devices
reading sources... [ 72%] draft
reading sources... [ 75%] eventloop
reading sources... [ 79%] index
reading sources... [ 82%] logging
reading sources... [ 86%] morethanbindings
reading sources... [ 89%] pyversions
reading sources... [ 93%] serialization
reading sources... [ 96%] ssh
reading sources... [100%] unicode

/home/docs/checkouts/readthedocs.org/user_builds/sakurai-youhei-pyzmq/envs/latest/lib/python3.7/importlib/__init__.py:127: VisibleDeprecationWarning: zmq.eventloop.minitornado is deprecated in pyzmq 14.0 and will be removed.
    Install tornado itself to use zmq with the tornado IOLoop.
    
  return _bootstrap._gcd_import(name[level:], package, level)

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/sakurai-youhei-pyzmq/envs/latest/lib/python3.7/site-packages/sphinx/cmd/build.py", line 280, in build_main
    app.build(args.force_all, filenames)
  File "/home/docs/checkouts/readthedocs.org/user_builds/sakurai-youhei-pyzmq/envs/latest/lib/python3.7/site-packages/sphinx/application.py", line 348, in build
    self.builder.build_update()
  File "/home/docs/checkouts/readthedocs.org/user_builds/sakurai-youhei-pyzmq/envs/latest/lib/python3.7/site-packages/sphinx/builders/__init__.py", line 299, in build_update
    len(to_build))
  File "/home/docs/checkouts/readthedocs.org/user_builds/sakurai-youhei-pyzmq/envs/latest/lib/python3.7/site-packages/sphinx/builders/__init__.py", line 311, in build
    updated_docnames = set(self.read())
  File "/home/docs/.pyenv/versions/3.7.3/lib/python3.7/contextlib.py", line 119, in __exit__
    next(self.gen)
  File "/home/docs/checkouts/readthedocs.org/user_builds/sakurai-youhei-pyzmq/envs/latest/lib/python3.7/site-packages/sphinx/util/logging.py", line 213, in pending_warnings
    memhandler.flushTo(logger)
  File "/home/docs/checkouts/readthedocs.org/user_builds/sakurai-youhei-pyzmq/envs/latest/lib/python3.7/site-packages/sphinx/util/logging.py", line 178, in flushTo
    logger.handle(record)
  File "/home/docs/.pyenv/versions/3.7.3/lib/python3.7/logging/__init__.py", line 1529, in handle
    self.callHandlers(record)
  File "/home/docs/.pyenv/versions/3.7.3/lib/python3.7/logging/__init__.py", line 1591, in callHandlers
    hdlr.handle(record)
  File "/home/docs/.pyenv/versions/3.7.3/lib/python3.7/logging/__init__.py", line 901, in handle
    rv = self.filter(record)
  File "/home/docs/.pyenv/versions/3.7.3/lib/python3.7/logging/__init__.py", line 762, in filter
    result = f.filter(record)
  File "/home/docs/checkouts/readthedocs.org/user_builds/sakurai-youhei-pyzmq/envs/latest/lib/python3.7/site-packages/sphinx/util/logging.py", line 417, in filter
    raise SphinxWarning(message)
sphinx.errors.SphinxWarning: autodoc: failed to import module 'ioloop' from module 'zmq.auth'; the following exception was raised:
No module named 'tornado'

Warning, treated as error:
autodoc: failed to import module 'ioloop' from module 'zmq.auth'; the following exception was raised:
No module named 'tornado'
```
</details>